dev/login-form.html

```diff
@@ -21,10 +69,46 @@
         padding: var(--lumo-space-l);
         display: flex !important;
       }
+
+      .controls {
+        display: grid;
+        grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
+        gap: 2rem;
+        margin-bottom: 2rem;
+        font-size: 0.875rem;
+      }
+
+      .control-group {
+        display: flex;
+        flex-direction: column;
+        gap: 1em;
+      }
     </style>
 
+    <div class="controls">
+      <div class="control-group">
+        <vaadin-checkbox-group label="Custom content">
+          <vaadin-checkbox id="footer" value="footer" label="Footer"></vaadin-checkbox>
+          <vaadin-checkbox id="custom-form-area" value="custom-form-area" label="Custom form element"></vaadin-checkbox>
+        </vaadin-checkbox-group>
+      </div>
+
+      <div class="control-group">
+        <vaadin-checkbox-group label="State toggle">
+          <vaadin-checkbox label="Error" id="error" value="error"></vaadin-checkbox>
+          <vaadin-checkbox label="No forgot password" id="noForgotPassword" value="noForgotPassword"></vaadin-checkbox>
+          <vaadin-checkbox label="Disabled" id="disabled" value="disabled"></vaadin-checkbox>
+        </vaadin-checkbox-group>
+      </div>
+      <div class="control-group">
+        <vaadin-radio-group id="direction" label="Direction">
+          <vaadin-radio-button label="LTR" value="ltr" checked></vaadin-radio-button>
+          <vaadin-radio-button label="RTL" value="rtl"></vaadin-radio-button>
+        </vaadin-radio-group>
+      </div>
+    </div>
     <div id="wrapper">
-      <vaadin-login-form no-autofocus></vaadin-login-form>
+      <vaadin-login-form no-autofocus> </vaadin-login-form>
```

Comment:
I guess whitespace change could be reverted?

Reply:
Thanks. Done.